app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def id; end
+
```

Comment:
- [ ] When you will open the `/users` route it will look for the `index` action in the users controller, Rails follows its naming conventions. Please fix this issue by changing the `id` method to `index` and always remember to follow the rails naming conventions.